components/Common/Card/index.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import { withThemeByDataAttribute } from '@storybook/addon-themes';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+import Card from './index';
+
+type Story = StoryObj<typeof Card>;
+type Meta = MetaObj<typeof Card>;
+
+export const Default: Story = {
+  args: {
+    title: 'Node.js March 17th Infrastructure Incident Post-mortem',
+    type: 'vulnerability',
+    subtitle: 'Vulnerabilities',
+    description:
+      'Starting on March 15th and going through to March 17th (with much of the issue being mitigated on the 16th), users were receiving intermittent 404 responses when trying to download Node.js from nodejs.org, or even accessing parts of the website.',
+    author: {
+      firstName: 'Hayden',
+      lastName: 'Bleasel',
+      src: 'https://avatars.githubusercontent.com/u/',
+    },
+    date: new Date(),
+  },
+  decorators: [
+    Story => (
+      <div className="max-w-lg">
+        <Story />
+      </div>
+    ),
+  ],
+};
+
+export const Dark: Story = {
+  ...Default,
+  decorators: [
+    ...(Default.decorators ?? []),
+    withThemeByDataAttribute({
+      themes: {
+        light: '',
+        dark: 'dark',
+      },
+      defaultTheme: 'dark',
+      attributeName: 'data-theme',
+    }),
+  ],
+};
```

Comment:
Dont need to add a story for the Dark mode. Chromatic gives output for light and dark mode

Reply:
You're right. I had kept two stories as the Common/Card component task described that it was necessary to have a story per each state, and mentioned dark mode as its own state.

However, what you just mentioned is true. There's no need to have two different stories as we have the theme dropdown.